stories/topics/proc-aws-deleting-backups-playbook.adoc

```diff
@@ -15,6 +15,11 @@ The use of `aws_backups_delete` is desscribed in this section.
 
 . Populate the `command_generator_data` directory with the configuration file template.
 +
+[NOTE]
+====
+On Linux, any file or directory created by the command generator will be owned by `root:root` by default. To change the ownership of the files and directories, you can run the `sudo chmod` command after the files are created. Check the xref:con-tech-note-linux-files-owned-by-root[technical notes] for more information.
```

Comment:
generator is owned by...

For more information, read xref:con-tech-note-linux-files-owned-by-root[technical notes].